### PR TITLE
faq, toc: add the TiDB in K8s FAQ doc

### PR DESCRIPTION
**Pull request by DanielZhangQD**

### CODE REVIEW

File: dev/faq/tidb-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: TiDB on Kubernetes 常见问题
```
Comment (lilin90) [inline dev/faq/tidb-kubernetes.md]:
```suggestion
title: Kubernetes 上的 TiDB 集群常见问题
```

File: dev/faq/tidb-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+category: FAQ
+---
+
+# TiDB on Kubernetes 常见问题
```
Comment (lilin90) [inline dev/faq/tidb-kubernetes.md]:
```suggestion
# Kubernetes 上的 TiDB 集群常见问题
```

File: dev/faq/tidb-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## 时区设置
+
+默认情况下，在 Kubernetes 集群上部署的 TiDB 集群各组件容器中的时区为 UTC，如果要修改时区配置，有下面两种情况：
+
+* 第一次部署集群
+
+    在 TiDB 集群 values.yaml 中，修改 timezone 配置，例如：timezone: Asia/Shanghai，然后部署 TiDB 集群。
```
Comment (lilin90) [inline dev/faq/tidb-kubernetes.md]:
```suggestion
    在 TiDB 集群的 `values.yaml` 文件中，修改 `timezone` 配置，例如：`timezone: Asia/Shanghai`，然后部署 TiDB 集群。
```

File: dev/faq/tidb-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* 集群已经在运行
+
+    如果 TiDB 集群已经在运行，需要做如下修改：
+    * 在 TiDB 集群 values.yaml 中，修改 timezone 配置，例如：timezone: Asia/Shanghai，然后升级 TiDB 集群
+    * 参考[时区支持](../how-to/configure/time-zone.md)，修改 TiDB 服务时区配置
```
Comment (lilin90) [inline dev/faq/tidb-kubernetes.md]:
```suggestion
    * 参考[时区支持](/how-to/configure/time-zone.md)，修改 TiDB 服务时区配置。
```

File: dev/faq/tidb-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+默认情况下，在 Kubernetes 集群上部署的 TiDB 集群各组件容器中的时区为 UTC，如果要修改时区配置，有下面两种情况：
+
+* 第一次部署集群
+
+    在 TiDB 集群 values.yaml 中，修改 timezone 配置，例如：timezone: Asia/Shanghai，然后部署 TiDB 集群。
+* 集群已经在运行
+
+    如果 TiDB 集群已经在运行，需要做如下修改：
+    * 在 TiDB 集群 values.yaml 中，修改 timezone 配置，例如：timezone: Asia/Shanghai，然后升级 TiDB 集群
```
Comment (lilin90) [inline dev/faq/tidb-kubernetes.md]:
```suggestion
    * 在 TiDB 集群 values.yaml 中，修改 timezone 配置，例如：timezone: Asia/Shanghai，然后升级 TiDB 集群。
```

**Comment by lilin90** [comment]
@DanielZhangQD After the comments are addressed, please also add the same changes to `v3.0`. Thanks!

**Comment by lilin90** [comment]
@DanielZhangQD Please add the file to `TOC.md` in both `dev` and `v3.0`. You can use this name: "Kubernetes 上的 TiDB FAQ"

**Comment by DanielZhangQD** [comment]
> @DanielZhangQD Please add the file to `TOC.md` in both `dev` and `v3.0`. You can use this name: "Kubernetes 上的 TiDB FAQ"

Done.

**Pull request merged**